README.md

```diff
@@ -1,2 +1,90 @@
-# ibm-cloud-env-golang
-IBM Cloud Environment for Go
+# IBM Cloud Environment
+
+The `ibm-cloud-env-golang` module allows to abstract environment variables from various Cloud compute providers, such as, but not limited to, CloudFoundry and Kubernetes, so the application could be environment-agnostic.
+
+The module allows to define an array of search patterns that will be executed one by one until required value is found.
```

Comment:
rephrase: The module allows for an array of search patterns that will be executed one by one until required value is found.